src/status_im2/contexts/wallet/select_address_to_watch/view.cljs

```diff
@@ -1,27 +1,59 @@
 (ns status-im2.contexts.wallet.select-address-to-watch.view
   (:require
-    [clojure.string :as string]
+    clojure.string
     [quo.core :as quo]
-    [quo.theme :as quo.theme]
     [re-frame.core :as re-frame]
-    [react-native.clipboard :as clipboard]
     [react-native.core :as rn]
     [react-native.safe-area :as safe-area]
     [reagent.core :as reagent]
+    [status-im2.constants :as constants]
     [status-im2.contexts.wallet.select-address-to-watch.style :as style]
+    [utils.debounce :as debounce]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
-(defn view-internal
+(defn- address-input
+  [input-value input-focused?]
+  (fn []
```

Comment:
I guess we can remove this `(fn [])` as we don't have any atoms.

Reply:
sure I'll take a look at removing that.

Reply:
Yes, that's true, the `fn []` is not needed. 

Actually, this `(fn[])` will cause problems with rerenders.

If `input-value`  or `input-focused?` change, the body of the function will not be re executed, and if the component rerenders (e.g. because a sub changes), `input-value` and `input-focused` will have the value they had when the component mounted, not the current one